server/migrations/20210719151134-make-matchmaking-preferences-based-on-type.js

```diff
@@ -0,0 +1,77 @@
+const sql = require('sql-template-strings')
+
+exports.up = async function (db) {
+  await db.runSql(`ALTER TABLE matchmaking_preferences ADD COLUMN data jsonb;`)
+  // This column is not used anymore so it's not necessary
+  await db.runSql(`ALTER TABLE matchmaking_preferences DROP COLUMN updated_at;`)
+
+  // This should probably be batched, maybe? Shouldn't be an issue with our current amount of users,
+  // I think.
+  const { rows: preferences } = await db.runSql(`
+    SELECT user_id, matchmaking_type, use_alternate_race, alternate_race
+    FROM matchmaking_preferences;
+  `)
+
+  await Promise.all(
+    preferences.map(p => {
+      const data = {
+        useAlternateRace: p.use_alternate_race,
+        alternateRace: p.alternate_race,
+      }
+
+      return db.runSql(sql`
```

Comment:
This can all be written as a single update query, without pulling all the data into JS-land first and creating 1000s of promises. Would vastly prefer that you do that.

Generally don't write migrations that need to operate on a bunch of data on the JS side unless you *have* to (and since this is just a simple modification on existing data in the table, that is definitely *not* the case here)